testapi.pm

```diff
@@ -690,23 +682,24 @@ sub assert_script_run {
     return;
 }
 
-=head2 script_sudo
+=head2 script_run
 
-  script_sudo($program [, $wait]);
+  script_run($program [, $wait]);
 
-Run C<$program> using sudo. Handle the sudo timeout and send password when appropriate.
-C<$wait_seconds> defaults to 2 seconds.
+Run C<$program> (by assuming the console prompt and typing it).
 
-I<The implementation is distribution specific and not always available.>
+The C<$wait> parameter will (unless 0) wait for the script to finish
+by following the script with an echo to serial line and waiting
+for it. Default timeout is 30s.
 
```

Comment:
`script_run` calls `$distri->script_run` thus I'm for to include
`I<The implementation is distribution specific and not always available.>` here as well.